content/docs/testing-recipes.md

```diff
@@ -278,126 +220,106 @@ it("changes value when clicked", () => {
 UIs could use timer based functions like `setTimeout` to run time sensitive code. In this example, a multiple choice panel waits for a selection and advances, timing out if a selection isn't made in 5 seconds.
 
 ```jsx
-function Card(props) {
-  useEffect(
-    props => {
-      const timeoutID = setTimeout(() => {
-        props.onSelect(null);
-      }, 5000);
-      return () => {
-        clearTimeout(timeoutID);
-      };
-    },
-    [props.onSelect]
```

Comment:
This gave a linting warning because we didn't include `props`, so I destructured `onSelect` instead.